build-tools/packages/version-tools/src/internalVersionScheme.ts

```diff
@@ -22,9 +22,14 @@ const MINIMUM_SEMVER_PRERELEASE_SECTIONS = 4;
 
 /**
  * The first part of the semver prerelease value is called the "prerelease identifier". For Fluid internal versions, the
- * value must always match this constant.
+ * value must always match one of these values.
+ *
+ * @privateRemarks
+ *
+ * "internal" should always be the first value since there is code that treats ALLOWED_PRERELEASE_IDENTIFIER[0] as the
+ * "default value", and internal was the default value when we started using the internal version scheme.
  */
-export const REQUIRED_PRERELEASE_IDENTIFIER = "internal";
+export const ALLOWED_PRERELEASE_IDENTIFIERS = ["internal", "rc"];
```

Comment:
Alternative suggestion, keep "internal" as its own string export, and reference it here? Code that cares about only that one could use the string export. Don't feel strongly about it.